WordPress/Docs/WP/GlobalVariablesOverrideStandard.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Global Variables Override"
+    >
+    <standard>
+    <![CDATA[
+    WordPress global variables should not be overridden. This includes variables imported with 
+    the `global` statement within functions, variables assigned via the `$GLOBALS` superglobal array,
+    and variables assigned in the global namespace.
+
+    A few select WordPress global variables like `$content_width` and `$wp_cockneyreplace` are 
```

Comment:
Since those are the two only WP global variables that can be overriden, I think this paragraph could be rephrased to remove the "like".